fabtests/include/shared.h

```diff
@@ -284,6 +277,13 @@ char *cnt_str(char str[FT_STR_LEN], long long cnt);
 int size_to_count(int size);
 size_t datatype_to_size(enum fi_datatype datatype);
 
+static inline int ft_use_size(int index, int enable_flags)
+{
+	return test_size[index].size <= fi->ep_attr->max_msg_size &&
+		((enable_flags == FT_ENABLE_ALL) ||
+		(enable_flags & test_size[index].enable_flags));
+}
```

Comment:
The dgram_pingpong test can be updated to remove its check for max_msg_size and rely on the above check instead.

Reply:
Good call. Updated